src/dq_suite/output_transformations.py

```diff
@@ -51,8 +51,22 @@ def create_parameter_list_from_results(result: dict) -> list[dict]:
     return [parameters]
 
 
+def get_grouped_ids_per_deviating_value(
+    filtered_df: DataFrame,
+    unique_identifier: list,
+    number_of_unique_ids: int,
```

Comment:
Het lijkt erop dat `number_of_unique_ids` (zie code hieronder) op r151 wordt aangemaakt en vervolgens alleen wordt meegegeven aan deze functie. 

In dat geval zou ik deze parameter van de functie verwijderen, en dan de definitie `number_of_unique_ids = len(unique_identifier)` in deze functie zelf zetten, omdat deze af te leiden valt uit de andere `unique_identifier` parameter.

Reply:
Klopt, hij wordt 1 keer aangemaakt buiten de loop omdat het constant is. We kunnen hem in de functie integreren voor leesbaarheid ten koste van milli beetje performance

Reply:
Idd, het ging me niet per se om de performance, maar vooral om het terugbrengen van de hoeveelheid koppeling tussen beide functies. Dat maakt het unit testen ook makkelijker, en zo isoleren we de logica.